### ISSUE TITLE
adbapi.ConnectionPool.noisy defaults to True, when it should be False

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1806]
|<img alt="moof's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| moof reported|
|-|-|
|Trac ID|trac#1806|
|Type|defect|
|Created|2006-06-08 10:23:40Z|

The documentation for adbapi.ConnectionPool.!__init!__ claims cp_noisy defaults to False, when it actually defaults to True.

It should default to false, as defaulting to True is a security risk: it prints the user name and password used to connect to the db in the log.

Attachments:

* [adbapi.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ade/adefdf1d5af74d813709062bc4c8260df5b485f7/0cd23300f76fa7f5b85a8f371d053173dd19c26a.patch) (545 bytes) - added by moof on 2006-06-08 10:25:05Z - Patch to set the default to False

<details><summary>Searchable metadata</summary>

```
trac-id__1806 1806
type__defect defect
reporter__moof moof
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1149762220000000 1149762220000000
changetime__1209475921000000 1209475921000000
version__None None
owner__moof moof
cc__therve
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve set owner to moof|
|-|-|

Let's say I reviewed it :).

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set status to `closed`|
|-|-|

(In [23460]) aChange default value of t.e.adbapi.ConnectionPool.noisy to False, as documented
and expected. This also removes a security risk as username and password were
printed in the logs.

Author: moof
Reviewer: therve
Fixes [#1806](https://github.com/twisted/twisted/issues/1806)
